src/main/java/frc/robot/RobotContainer.java

```diff
@@ -20,25 +29,53 @@
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
-  private final DrivetrainSubsystem m_drivetrainSubsystem = new DrivetrainSubsystem();
 
   private final Joystick m_jsDriver = new Joystick(0);
+  private final Joystick m_jsDriverSteer = new Joystick(1);
+
+  final Controls controls = new Controls(m_jsDriver, m_jsDriverSteer);
+
+  private final SwerveDrive m_driveTrain = SwerveDrive.GetInstance();
+
+  SendableChooser<SequentialCommandGroup> autoChooser = new SendableChooser<>();
+
+  // Tab to display autonous data
+  public static final ShuffleboardTab autoTab = Shuffleboard.getTab("Autonomous");
+  // Entries for auto data
+  public static final GenericEntry 
```

Comment:
robot container is not the best place for public static objects. should move to their own singleton or static class.